### DISCUSSION TITLE
Allow HTTP URLs for `dark_logo` and `light_logo`

### DISCUSSION THREAD

**Discussion by IrrerPolterer** [pradyunsg/furo#772] [Ideas] [answered]
### Status Quo:

The `html_theme_options` parameter allows configuration of the logo using locally stored image files in the static directory, like so:

```python
html_static_path = ["_static"]
html_theme_options = {
    "light_logo": "my-logo-light.png",
    "dark_logo": "my-logo-dark.png",
}
``` 

This works well, assuming the image files are stored in the `_static` directory.

### Problem Statement:

I tend to store things like logos, favicons and related resources in a cloud storage solution, where they are accessible by various services. This avoids the need to store copies of these files along side the code of each of these applications.

I would like to specify a remote URL for the logos, to make use of my centralized resource storage. But currently, these configuration options only support locally stored resource paths.

### Suggestion:

In addition to local static resource paths, allow for HTTP URLs to be used to specify logo images. Example:

```python
html_theme_options = {
    "light_logo": "https://example.org/storage/my-logo-light.png",
    "dark_logo": "https://example.org/storage/my-logo-dark.png",
}
```

**Comment by pradyunsg** [comment]
Seems reasonable -- could you file a trackable task issue for this? :)

**Reply by IrrerPolterer** [reply, marked as answer]
Will do!
